crates/router/tests/connectors/cybersource.rs

```diff
@@ -185,7 +169,11 @@ async fn should_fail_payment_for_invalid_exp_month() {
     let x = response.response.unwrap_err();
     assert_eq!(
         x.message,
-        r#"Declined - One or more fields in the request contains invalid data [{"field":"paymentInformation.card.expirationMonth","reason":"INVALID_DATA"}]"#,
+        r#"Declined - One or more fields in the request contains invalid data"#,
```

Comment:
```suggestion
        "Declined - One or more fields in the request contains invalid data",
```

This can be a regular string literal.